plugin/web/extensions/sidebar/core/components/mobile/hooks/projectHooks.ts

```diff
@@ -204,6 +211,41 @@ export default ({
     [processOverrides],
   );
 
+  const handleOverrideForMobileBuildingSearch = useCallback(
+    ({ dataID, overrides }: { dataID: string; overrides: any }) => {
+      const mergedOverrides = merge(
+        {},
+        cachedDatasetOverrides.current.get(dataID) ?? {},
+        overrides ?? {},
+      );
+      postMsg({
+        action: "updateDatasetInScene",
+        payload: { dataID, overrides: mergedOverrides },
+      });
+    },
+    [],
+  );
+
+  const handleOverrideForMobileBuildingSearchClose = useCallback(
+    ({ dataID }: { dataID: string }) => {
+      const overrides = merge(
+        {
+          "3dtiles": {
+            show: true,
+            color: "white",
+          },
+        },
+        cachedDatasetOverrides.current.get(dataID) ?? {},
+      );
+
+      postMsg({
+        action: "updateDatasetInScene",
+        payload: { dataID, overrides },
+      });
+    },
+    [],
+  );
```

Comment:
You aren't doing any sort of "cleanseOverride" kind of thing here. Are ALL override values that the building search changes values that will by default have values?

Reply:
buildingSearch will and always update `show` & `color`，this is the same as the cleanseOverride for `field` building search on desktop. 
Since on mobile building search overrides become independent and above dataset override systems, it can't join dataset cleanseOverride calculation, so the value here is used as a default value for 3d tiles buildings. 
Without this default value it will have some problem, for example building search filtered the buildings, and when building search close, the cached overrides from dataset does not contain `show`, and the `show` conditions will be left there.